pynucastro/rates/rate.py

```diff
@@ -790,6 +790,11 @@ def invert(self):
                                max_products=self.max_reactants)
         return newfilter
 
+class ReacLibLibrary(Library):
+
+    def __init__(self, libfile='20180319default2', rates=None, read_library=True):
+        assert(libfile == '20180319default2'  and rates == None and read_library == True), "Only the 20180319default2 default ReacLib snapshot is accepted"
```

Comment:
an `assert` should not use parantheses
https://stackoverflow.com/questions/13390401/design-of-python-why-is-assert-a-statement-and-not-a-function